File: Act-1-POO/Act-1-POO-PHP/Pelicula.php
<?php

class Pelicula {
	private $IdPelicula;
	private $NombreDePelicula;
	private $DirectorDePelicula;
	private $FechaDeEstreno;
	private $DuracionPelicula;
	private $NombreGenero;
	private $DescripcionGenero;

	// Constructor
	public function __construct($id, $nombre, $director, $fecha, $duracion, $genero, $descripcion) {
		$this->IdPelicula = $id;
		$this->NombreDePelicula = $nombre;
		$this->DirectorDePelicula = $director;
		$this->FechaDeEstreno = $fecha;
		$this->DuracionPelicula = $duracion;
		$this->NombreGenero = $genero;
		$this->DescripcionGenero = $descripcion;	
	}

	// Getters
	public function getIdPelicula() {
		return $this->IdPelicula;
	}

	public function getNombreDePelicula() {
		return $this->NombreDePelicula;
	}

	public function getDirectorDePelicula() {
		return $this->DirectorDePelicula;
	}

	public function getFechaDeEstreno() {
		return $this->FechaDeEstreno;
	}

	public function getDuracionPelicula() {
		return $this->DuracionPelicula;
	}

	public function getNombreGenero() {
		return $this->NombreGenero;
	}

	public function getDescripcionGenero() {
		return $this->DescripcionGenero;
	}

	// Setters
	public function setIdPelicula($id) {
		$this->IdPelicula = $id;
	}

	public function setNombreDePelicula($nombre) {
		$this->NombreDePelicula = $nombre;
	}

	public function setDirectorDePelicula($director) {
		$this->DirectorDePelicula = $director;
	}

	public function setFechaDeEstreno($fecha) {
		$this->FechaDeEstreno = $fecha;
	}

	public function setDuracionPelicula($duracion) {
		$this->DuracionPelicula = $duracion;
	}

	public function setNombreGenero($genero) {
		$this->NombreGenero = $genero;
	}

	public function setDescripcionGenero($descripcion) {
		$this->DescripcionGenero = $descripcion;
	}
}

?>

File: Act-1-POO/Act-1-POO-Phyton/Pelicula.py
class Pelicula:
	def __init__(self, id_pelicula, nombre_de_pelicula, director_de_pelicula, 
		fecha_de_estreno, duracion_pelicula, nombre_genero, 
		descripcion_genero):
		self.id_pelicula = id_pelicula
		self.nombre_de_pelicula = nombre_de_pelicula
		self.director_de_pelicula = director_de_pelicula
		self.fecha_de_estreno = fecha_de_estreno
		self.duracion_pelicula = duracion_pelicula
		self.nombre_genero = nombre_genero
		self.descripcion_genero = descripcion_genero

pelicula = Pelicula(
	id_pelicula=1,
	nombre_de_pelicula="El Padrino",
	director_de_pelicula="Francis Ford Coppola",
	fecha_de_estreno="1972-03-24",
	duracion_pelicula="2h 55m",
	nombre_genero="Crimen",
	descripcion_genero="Una saga sobre la familia Corleone."
)

print(pelicula.nombre_de_pelicula)
